features/meta.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: <meta>
-description: The `<meta>` element TODO.
+description: The `<meta>` element represents metadata that cannot be represented by other HTML meta-related elements.
```

Comment:
This description mainly says what `<meta>` isn't, not what it is. What's the most common use of `<meta>` and can we put that in the description?